Sources/XCMetricsCommon/UploadRequestExtraInfo.swift

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2020 Spotify AB.
```

Comment:
I created a new Shared module between the client and the backend to held the models used by both. At the moment, it only contains this one. But I plan to refactor in the future to move more things here and avoid code duplication.